### PR TITLE
Fixed #33778 -- Updated build-system configuration.

### PR DESCRIPTION
**Pull request by badziyoussef**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 from setuptools import setup
 
+sys.path.insert(0, ".")
```
Comment (felixxm) [inline setup.py:8]: Why we need this?

Comment (badziyoussef) [reply setup.py:8]: The first workflow failed with error on import django so I included it in path

Comment (felixxm) [reply setup.py:8]: Editable install should just work according the comments in ticket-33778 :thinking: (\cc @mgorny)

Comment (badziyoussef) [reply setup.py:8]: mI agree, I will remove the line and see if workflow succeed

Comment (mgorny) [reply setup.py]: I agree, I don't see why `sys.path` manipulation would be needed there.

Comment (mohamedTbarka) [reply setup.py]: @mgorny, why do u see sys.path not needed ??

Comment (mgorny) [reply setup.py]: Because I don't see any `import` statements below it.

**Comment by badziyoussef** [comment]
@felixxm PR seems good to you now?

**Comment by felixxm** [comment]
> @felixxm PR seems good to you now?

We also should update [_"Packaging your app"_](https://docs.djangoproject.com/en/4.1/intro/reusable-apps/#packaging-your-app) docs.

**Comment by badziyoussef** [comment]
> > @felixxm PR seems good to you now?
> 
> We also should update [_"Packaging your app"_](https://docs.djangoproject.com/en/4.1/intro/reusable-apps/#packaging-your-app) docs.

doc updated can you recheck the PR?

**Comment by felixxm** [comment]
@badziyoussef Thanks, looks good :+1: Welcome aboard :boat:

**Pull request merged**